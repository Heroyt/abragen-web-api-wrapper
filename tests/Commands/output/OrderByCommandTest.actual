Column of orderBy command is supposed to be string, instance of stdClass given