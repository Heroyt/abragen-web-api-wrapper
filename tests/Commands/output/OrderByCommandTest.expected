Column of orderBy command is supposed to be string, intance of stdClass given